src/basics.php

```diff
@@ -151,3 +151,28 @@ function breakpoint()
         );
     }
 }
+
+if (!function_exists('dd')) {
+    /**
+     * Prints out debug information about given variable and dies.
+     *
+     * Only runs if debug mode is enabled.
+     * It will otherwise just continue code execution and ignore this function.
+     *
+     * @param mixed $var Variable to show debug information for.
+     * @param bool|null $showHtml If set to true, the method prints the debug data in a browser-friendly way.
+     * @return void
+     * @link http://book.cakephp.org/3.0/en/development/debugging.html#basic-debugging
+     */
+    function dd($var, $showHtml = null)
+    {
+        if (!Configure::read('debug')) {
+            return;
+        }
+
+        debug($var, $showHtml, false);
+        $backtrace = debug_backtrace(false, 1);
```

Comment:
Couldn't these 2 lines added to function `debug()` instead?
Would that make @ADmad happy?